content/SCALE/SCALETutorials/SystemSettings/UpdateSCALE.md

```diff
@@ -9,60 +9,87 @@ tags:
+For information on how to file an issue ticket see [Filing an Issue Ticket in SCALE]({{< relref "FileIssueSCALE.md" >}}).
 {{< /hint >}}
 
-The TrueNAS SCALE **Update** screen lets users update their system using two different methods: manual or automatic.
-
-We recommend updating TrueNAS when the system is idle (no clients connected, no disk activity, etc). Most updates require a system reboot. 
-
+The TrueNAS SCALE **Update** screen provides users with two different methods to update the system, automatic or manual. 
+We recommend updating SCALE when the system is idle (no clients connected, no disk activity, etc.). 
+The system restarts after an upgrade. 
 Update during scheduled maintenance times to avoid disrupting user activities.
 
 {{< hint type=important >}}
-All auxiliary parameters are subject to change between major versions of TrueNAS due to security and development issues. We recommend removing all auxiliary parameters from TrueNAS configurations before upgrading.
+All auxiliary parameters are subject to change between major versions of TrueNAS due to security and development issues. 
+We recommend removing all auxiliary parameters from TrueNAS configurations before upgrading.
 {{< /hint >}}
 
-![UpdateTrainSCALE](/images/SCALE/22.02/ScaleUpdateTrain.png "SCALE Update Train")
+{{< trueimage src="/images/SCALE/23.10/SystemUpdateScreenAvailableUpdate.png" alt="System Update Upgrade Available" id="1: System Update Upgrade Available" >}}
+
+## Performing an Automatic Update
+
+If an update is available, click **Apply Pending Update** to install it. 
+
+The **Save configuration settings from this machine before updating** window opens.
+
+{{< trueimage src="/images/SCALE/23.10/SaveConfigSettingsWindow.png" alt="Save Configuration Settings" id="2: Save Configuration Settings" >}}
 
-## Automatic
+Click **Export Password Secret Seed** then click **Save Configuration**. 
+The **Apply Pending Updates** window opens. 
 
-Select the **Check for Updates Daily and Download if Available** option to automatically download updates.  
+{{< trueimage src="/images/SCALE/23.10/ApplyPendingUdates.png" alt="Apply Pending Updates" id="3: Apply Pending Updates" >}}
 
-If an update is available, click **Apply Pending Update** to install it.
+Click **Confirm**, then **Continue** to start the automatic installation process. 
+TrueNAS SCALE downloads the configuration file and the update file, then starts the install. 
 
-## Manual
+## Performing a Manual Update
 
-To do a manual update, click **Download Updates** and wait for the file to download to your system. 
+If the system detects an available update, to do a manual update click **Download Updates** and wait for the file to download to your system. 
 
-Download the [SCALE Manual Update File](https://www.truenas.com/download-truenas-scale/).
+You can also download the [SCALE Manual Update File](https://www.truenas.com/download-truenas-scale/) from the website.
```

Comment:
I think downloading from elsewhere is probably the more likely route and choosing to download and then update rather than pressing apply pending should be presented as the option.  It also makes the first sentence less clunky.

```suggestion
To do a manual update, download the [SCALE Manual Update File](https://www.truenas.com/download-truenas-scale/) from the website.

If the system detects an available update, you can also click **Download Updates** and wait for the file to download to your system.
```